clmm/modbackend/ccl.py

```diff
@@ -0,0 +1,164 @@
+# Functions to model halo profiles 
```

Comment:
Can you add docstrings to the fucntions here? If the API here work exactly as in `clmm/modbackend/func_layer.py`, you could just copy them.

Reply:
In this file we have the OO interface to CCL, thus, it will inherit the docstring from CLMModeling, for this reason it is better to put all documentation in clmm_modeling.py and only add a docstring to the methods in ccl.py when we want to overwrite the father's documentation.

For example, lets say that in clmm_modeling.py we add a docstring documenting the behavior of set_mass, since CCLCLMModeling is a child of CLMModeling its implementation of set_mass will inherit the docstring from CLMModeling's set_mass method. If you want to see this working just use:

```
import inspect
print(inspect.getdoc(moo.set_mass))
```

and you will see the docstring that I just added to set_mass in clmm_modeling.py.

Since the documentation in func_layer.py concerns the functional interface they will be a bit different from the OO documentation. But here we have the same as in the oo interface, we just need to document the func_layer once and all back-ends will inherit it.